webdriverio_taf/po/pages/shop.page.js

```diff
@@ -0,0 +1,79 @@
+import ProductCard from '../components/productCard.component'
+import Cart from '../components/cart.component'
+import ProductRowInCart from '../components/productRowInCart.component'
+import SizeFilter from '../components/sizeFilter.component'
+import { step } from '../../support/helpers'
+
+class ShopPage {
+  constructor (browser) {
+    this.url = '/'
+    this.browser = browser
+    this.cart = new Cart(browser)
+  }
+
+  get productCards () {
+    return this.browser.$$('[class^="Product__Container"]')
+  }
+
+  get toggleCartButton () {
+    return this.browser.$('button[class^="Cart__CartButton"]')
+  }
+
+  open () {
+    return step(`Navigate to shop page "${this.url}"`, async () => {
+      await this.browser.url(this.url)
+    })
+  }
+
+  getProductCard (productName) {
+    return new ProductCard(this.browser, productName)
+  }
+
+  getProductRowInCart (productName) {
+    return new ProductRowInCart(this.browser, productName)
+  }
+
+  async getProductCounterText () {
+    return await step('Get product counter text', async () => {
+      const locator = this.browser.$('*=Product(s) found')
+      const textContent = await locator.getText()
+      const match = textContent.match(/\d+/)
+      const number = match ? parseInt(match[0], 10) : null
+      return number
```

Comment:
If this method is named `get*Text` it should not return `null`
```suggestion
      return match ? parseInt(match[0], 10) : ''
```
or be renamed it to `get*TextOrNull`